src/analysis/special_functions/pow.lean

```diff
@@ -169,16 +184,19 @@ begin
   exact is_closed_eq continuous_fst continuous_const,
 end
 
-lemma continuous_at_const_cpow {a b : ℂ} (ha : a ≠ 0) : continuous_at (cpow a) b :=
+/- Continuity of `λ x, a ^ x`: union of these two lemmas is optimal. -/
+
+lemma continuous_at_const_cpow {a b : ℂ} (ha : a ≠ 0) : continuous_at (λ x, a ^ x) b :=
 begin
-  have cpow_eq : cpow a = λ b, exp (log a * b),
-    by { ext1 b, rw [cpow_eq_pow, cpow_def_of_ne_zero ha], },
+  have cpow_eq : (λ x:ℂ, a ^ x) = λ x, exp (log a * x),
+    by { ext1 b, rw [cpow_def_of_ne_zero ha], },
   rw cpow_eq,
   exact continuous_exp.continuous_at.comp (continuous_at.mul continuous_at_const continuous_at_id),
 end
 
-lemma continuous_at_const_cpow' {a b : ℂ} (h : b ≠ 0) : continuous_at (cpow a) b :=
+lemma continuous_at_const_cpow' {a b : ℂ} (h : b ≠ 0) : continuous_at (λ x, a ^ x) b :=
 begin
+  simp_rw ←cpow_eq_pow,
```

Comment:
Instead of this, could you fix `zero_cpow_eq_nhds` to use the canonical power function instead of `cpow`? (And also `cpow_eq_nhds` while you're at it?)